app/controllers/tag_controller.rb

```diff
@@ -251,6 +251,15 @@ def widget
     render layout: false
   end
 
+  def blog2
+    nids = Tag.find_nodes_by_type(params[:id], 'note', nil).collect(&:nid)
+    @pagy, @notes = pagy(Node.where('status = 1 AND nid in (?)', nids).order('created DESC'), items: 6)
+    @tags = Tag.where(name: params[:id])
+    @tagnames = @tags.collect(&:name).uniq! || []
+    @title = @tagnames.join(',') + ' Blog' if @tagnames
+    @alltags = Tag.first(6)
```

Comment:
This looks really great - i'm just going to make a small suggested change here so we can set the featured topics manually. Thanks, @Cadreia !!!!

```suggestion
```